### ISSUE TITLE
Create Specific Tensorflow pet identification on Desktop

### ISSUE THREAD

**Issue by brandonC1234** [JasonDCox/ML-Mentorship-GovSchool#29]
## Description
Create a Tensorflow model in which I can upload images of my dogs with annotations to a folder and it will train to identify the different dogs. This model with work with the first one to account for the large amounts of false positives it will create from limited data input.

## Acceptance Criteria
 - Create a tensorflow model that can identify who my dog is using the webcam on my dekstop
 - Be able to load it with the general model

## References
 - https://www.youtube.com/watch?v=yqkISICHH-U&t=11373s

**Comment by brandonC1234** [comment]
## Progress
I have a system where you take images through your webcam and use the provided annotation software to annotate them. You then have to change the label map accordingly and create a TFrecord with it. This allows you to train the model.

## Performance
It does detect the dog, but it has an issue where it gives a lot of false positives. To address this, I will be crosschecking the results with a model trained to detection general cats and dogs.

**Comment by brandonC1234** [comment]
## Changes
I changed to the strategy of having a class for each breed of dog and a class for the unique dog.

## Progress
After 74,000 training steps, the model is complete and works properly

## Performance
The performance and metrics for the model has been uploaded in the Tensorflow folder in the metrics_results folder

FPS and memory usage were not measured as they are not important to the project since this is being run on the desktop rather than the Jetson.
